elated-egrets/bangazon-internal-site
# Ry employee looky loo

## DESCRIPTION

## How to pull down
Commands

```
git fetch --all
git checkout ryEmployeeLookyLoo
```

## Steps to test

1. Start virtual environment
1. Make sure Django server is running
1. got to http://127.0.0.1:8000/employees/1/
1. from the root directory run python manage.py test and see if the tests pass

## What you changed
Added employee details
## What was added to read me
Nothing
## Final Checklist
Before merging.
1. Has each file changed been reviewed?
2. Has the readme been updated to reflect changes made?
3. Has a teammate approved the PR?

Then Ship It!
